.travis.yml

```diff
@@ -3,5 +3,3 @@ node_js:
 - '6'
 - '5'
 - '4'
```

Comment:
I think these need to change also, we should only support 10, 12, and 14.

Reply:
Oh sorry about this misunderstanding. Added it in a dedicated commit https://github.com/emberjs/eslint-plugin-ember-internal/pull/22/commits/58ecb3d5b93f086db1eaee8a67559e0c259e956d